src/views/ImportFlow.vue

```diff
@@ -11,6 +10,7 @@
               </h2>
               <p>{{ 'flow-builder.create-flow-from-json' | trans }}</p>
               <div
+                v-if="trans('flow-builder.import-note')"
```

Comment:
@jcbashdown  Can you help me understand why we are checking for the import-note's presence?  Is this a dynamically populated string?

Reply:
I'm not sure this is a great way to do it but basically, I'm trying to allow for this message being empty. When embedded in voto5 the default message doesn't make sense. The intention is to allow overriding with an empty string (and embedding app should be able to override and localisation) but the element is still rendered if this happens. See here:

https://bitbucket.org/voto/voto5/pull-requests/1899/vmo-4246-integration-test-branch#Lresources/assets/js/floip/builder/routes/index.jsT26